### PR TITLE
Update jenkins-job-builder doc

### PR DESCRIPTION
**Pull request by toabctl**
- update command to upload jobs. the template dir
  must be given on the internal jenkins server
- mention that new jobs should always be defined in yaml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,7 +72,13 @@ jenkins/track-upstream-and-package.pl syntax OK
...
+in yaml format.
 To update jobs which land on ci.opensuse.org, run:
 
-   jenkins-jobs --ignore-cache update scripts/jenkins/jobs-obs/
+    jenkins-jobs --ignore-cache update scripts/jenkins/jobs-obs/
+
+To update jobs which land on the SUSE internal CI, run:
+
+    jenkins-jobs --ignore-cache update \
+        scripts/jenkins/jobs-ibs/:scripts/jenkins/jobs-ibs/templates/
```
Comment (nkrinner) [inline README.md]: The commands fail for me:  
```
jenkins_jobs.errors.JenkinsJobsException: A valid configuration file is required when not run as a test
/etc/jenkins_jobs/jenkins_jobs.ini is not a valid .ini file
```
Shouldn't we reference the config file (I assume scripts/jenkins/jenkins_jobs.ini.sample is the template for that) somewhere here?

File: README.md [hunk 2]
```diff
@@ -72,7 +72,13 @@ jenkins/track-upstream-and-package.pl syntax OK
 # jenkins jobs
 There are manually maintained jobs and some jobs are now using
 [jenkins-job-builder](http://docs.openstack.org/infra/jenkins-job-builder/)
-which defines jobs in yaml format.
+which defines jobs in yaml format. New jobs should always be defined
+in yaml format.
 To update jobs which land on ci.opensuse.org, run:
```
Comment (nkrinner) [inline README.md]: Skip the ```which land``` here? Makes more sense to me. Or I am misunderstanding the topic.

File: README.md [hunk 3]
```diff
@@ -72,7 +72,13 @@ jenkins/track-upstream-and-package.pl syntax OK
...
 
-   jenkins-jobs --ignore-cache update scripts/jenkins/jobs-obs/
+    jenkins-jobs --ignore-cache update scripts/jenkins/jobs-obs/
+
+To update jobs which land on the SUSE internal CI, run:
```
Comment (nkrinner) [inline README.md]: Skip the ```which land``` here?

File: README.md (hunk 2 above)
Comment (toabctl) [inline README.md]: Done

File: README.md (hunk 3 above)
Comment (toabctl) [inline README.md]: Done

File: README.md (hunk 1 above)
Comment (toabctl) [inline README.md]: Done

**Comment by nkrinner** [comment]
+1 and thanks for the changes!

**Comment by rsalevsky** [comment]
+1

Just one general improvement what I can see is that we should thought about to move the jjb yaml's to another folder. For me the scripts folder makes not really sense.

**Comment by toabctl** [comment]
> For me the scripts folder makes not really sense.

I agree. But imo out of scope for this PR.

**Pull request merged**